webpack.config.js

```diff
@@ -0,0 +1,29 @@
+var path = require('path');
+
+module.exports = {
+	entry: './src/app.js',
+	output: {
+		filename: 'app.js',
+		path: path.resolve(__dirname, 'dist')
+	},
+	target: 'node',
+	module: {
+		loaders: [{
+			test: /\.js$/,
+			use: require.resolve('./remove-hashbang-loader')
+		}]
+	},
+	externals: {
+		'mkfifo': 'commonjs mkfifo',
+		'sqlite3': 'commonjs sqlite3',
+		'mysql2': 'null',
```

Comment:
In the end this didn't work (webpack still tried and couldn't resolve the modules) - so I switched to a function that just usess `commonjs <modulename>` - this will cause the calls to require these modules to remain unchanged which is what we want after all.